core/unit_test/TestBitManipulation.cpp

```diff
@@ -463,8 +463,14 @@ constexpr bool test_byteswap2() {
   static_assert(byteswap<int8_t>(0x12) == 0x12);
   static_assert(byteswap<int16_t>(0x1234) == 0x3412);
   static_assert(byteswap<int32_t>(0x12345678) == 0x78563412);
+
+// FIXME_NVCC This case is overflowing int65_t and the conversion from uint64_t
+// to int64_t here is implementation-defined. This seems to be the reason for
+// the check to fail for nvcc with CUDA <= 11.3.0
```

Comment:
The comment is inaccurate. For one I donor think it is an overflow, it is a compiler bug.  It is not supposed to be a conversion, it is specifying the type and the underlying bit representation.
And it works with 11.3.0 so it is version less (not version less equal)